### DISCUSSION TITLE
Create iPhone version of the app?

### DISCUSSION THREAD

**Discussion by tomhammond** [fhdm-dev/scoppy#60] [General]
Hello, would it be possible to create an iPhone version of the app?  There are a lot of Apple users out there that are missing out on this wonderful project!  Apple phones have a USB converter cable that should work for that.  Thanks for considering this.

**Comment by fhdm-dev** [comment]
Hi,

Thanks for the suggestion. There's still so many features that I want to add to the firmware and Android app that unfortunately I don't have the time to work on an iPhone version at the moment. But it's definitely on my to-do list so it might happen one day :smile:

**Reply by tomhammond** [reply]
No problem, I understand!  I have an Android phone, but realize there are lots of Apple phone users that would love to have a Pico scope on their phones, too.   Glad that it's on the eventual to-do list.  :)
